app/ts/components/SendEthPage.tsx

```diff
@@ -48,40 +44,61 @@ const Main = () => {
 						<SendGuide />
 					</div>
 					<div class='[grid-area:controls]'>
-						<SendActions />
+						<FormActions />
 					</div>
 				</FormLayout>
 			</SendForm>
-		</TransferProvider>
+		</ErrorBoundary>
 	)
 }
 
+const SendForm = ({ children }: { children: ComponentChildren }) => {
+	const accountStore = useAccountStore()
+	const transfer = useTransferStore()
+
+	const handleSubmit = (event: Event) => {
+		event.preventDefault()
+
+		if (!accountStore.value.isConnected) {
+			accountStore.value.connectMutation.dispatch()
+			return
+		}
+
+		if (!transfer.value.isSigned) {
+			transfer.value.transactionResponseQuery.dispatch()
+			return
+		}
+
+		window.location.href = `#tx/${transfer.value.transactionResponse.hash}`
+		return
+	}
+
+	return <form onSubmit={handleSubmit}>{children}</form>
+}
+
 const SendGuide = () => {
-	const { asyncResponse } = useTransfer()
-	const [transactionResponse] = asyncResponse
+	const transferStore = useTransferStore()
 
-	switch (transactionResponse.state) {
+	if (transferStore.value.isSigned) return <Guide title='Request Successfully Sent!' content="Your transaction is awaiting confirmation from the chain. You may click on the View Transaction button to check it's status." />
```

Comment:
I believe we should be updating the transaction status all the time without user input?

Reply:
There is a separate page to view and track transaction status, would you suggest we automatically direct the user to that page?

Reply:
I mean instead of asking user to click View to see the status, you could show the status directly

Reply:
created a separate feature issue as i have some additions to add and test to the transitioned view. #68